core/json/src/main/scala/net/liftweb/json/Extraction.scala

```diff
@@ -319,14 +331,54 @@ object Extraction {
       }
     }
 
+    def newTuple(root: JValue, mappings: List[Mapping]): Any = {
```

Comment:
Ah, sorry, no, the pattern here is that we nest the `newX` functions inside of the larger extraction handler. So, visibility modifiers aren't allowed here because we're not in the class scope (and nothing outside this function can access it anyway).

At some point I'd like to unwind that behavior, but now is not the right time to do that.